### ISSUE TITLE
SQL Results in "location: besides" mode behaves oddly between VSCode restarts

### ISSUE THREAD

**Issue by ransagy** [mtxr/vscode-sqltools#211]
**Describe the bug**
When using the SQL results panel in the "location: besides" mode, where the panel opens "inside" of the SQL tab executing the query, Works well on its own. When closing and reopening VSCode, the SQL tab is retained (as per VSCode behavior/settings) and while the **space** that the SQL Results panel used to be in is retained, the panel is not. Executing another piece of SQL from that tab opens a new panel, leaving the empty space behind until manually closed.

**To Reproduce**
1. Configure SQLTools to work in location-besides mode via the settings json.
2. Open new SQL tab and connect to a DB.
3. Write some simple query SQL and execute it.
4. Confirm the SQL results panel opened besides the SQL tab.
5. restart VSCode.
6. Connect to a DB again in the retained SQL tab and execute a piece of SQL again.
7. The SQL Results panel opens in a new location, leaving the former space empty.

**Expected behavior**
The SQL Results panel will appear in the same location as before the restart.

**Desktop (please complete the following information):**
 - SQLTools Version: v0.17.18
 - VSCode Version: 1.33.1 (user setup)
 - OS: Windows 10 x64 10.0.17763

**Comment by mtxr** [comment]
I've fixed this. Also I took this shot to enhance this behavior.

Now the names were changed to next, current, end or a number.
next means 'next to active editor', so it will pick the next column on right closer to the active editor.
end means 'always the last column on right', so it will create a new column every time in the last position.
current is the current column.

This is the same behavior `vscode-mssql` is using.

The number is the number of the column the user want it to open, I believe this wont be widely used, but it was a suggestion in the vscode-mssql repo, so I decide to add it here as well.

Thanks @ransagy !
